pkg/providers/v2/interface.go

```diff
@@ -39,6 +46,71 @@ var _ cloudprovider.Interface = (*cloud)(nil)
 
 // cloud is the AWS v2 implementation of the cloud provider interface
 type cloud struct {
```

Comment:
Line 36 above needs to be updated to call newCloud as part of `RegisterCloudProvider`